### PR TITLE
Feat: Structuring Multiple Layout

### PR DESCRIPTION
**Pull request by rudestewing**

### CODE REVIEW

**Comment by insomnius** [comment]
Canggih, btw ada conflict.

**Comment by rudestewing** [comment]
sudah di fix cek lagi.

File: components/Footer.js [hunk 1]
```diff
@@ -4,18 +4,17 @@ const Footer = (props) => {
...
-                        <a className="no-underline hover:no-underline font-bold text-2xl lg:text-4xl text-gradient gradient" href="#">
-                            <a className="flex no-underline hover:no-underline" href="/">
-                                <div className="inline mx-1 col-start-1 my-auto">
-                                    <img src="/icons/logo.png" alt="Coronator Logo" className="h-6 w-6" />
-                                </div>
-                                <div className="inline mx-1 my-auto col-start-1">
-                                    <p className="text-gray-100 font-medium">Coronator</p>
-                                </div>
-                            </a>
+                        <div className="no-underline hover:no-underline font-bold text-2xl lg:text-4xl text-gradient gradient"  href="#">
```
Comment (insomnius) [inline components/Footer.js]: Pake html formatter dong @rudestewing , atau emang formatter lu gini?

**Comment by insomnius** [comment]
Anyway merging this dludeh

**Review by insomnius — APPROVED** [review]

**Pull request merged**